ocw/lib/gce.py

```diff
@@ -207,3 +209,112 @@ def cleanup_all(self):
                 if "warnings" in response:
                     for w in response["warnings"]:
                         self.log_warn(w["message"])
+        self.cleanup_vault_keys()
+
+    def cleanup_vault_keys(self):
+        self.log_info("## GCE Vault key cleaning - %s ##" % self.__project)
+        credentials = service_account.Credentials.from_service_account_info(
```

Comment:
```
credentials = service_account.Credentials.from_service_account_info(
            self.__credentials.getPrivateKeyData()
        )
        service = googleapiclient.discovery.build('iam', 'v1', credentials=credentials)
```

please drop this and just use https://github.com/SUSE/pcw/pull/125/files#diff-0dfc4095c73002d356e289d8edf872dcb8a6624ba9adc082c0c331aa9789bb9cR22 instead . you duplicating the code

Reply:
i did that in the very beginning and it was failing. i didnt overthink and i left it when i see that it was working. if i fix the rest i might take a look. but until then i am not going to change it as i am afraid that it will require additional changes. i have to investigate why it wasnt working and what changes it needs

Reply:
sure thing it is up to you to decide the order , just saying that we will need to revisit this before merging in any case or find really good reasons to keep this duplication